frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/topology/components/nodeStatusIcons.css

```
@@ -21,6 +21,16 @@
   fill: #e0182d;
 }
 
+.sync {
+  color: `#0066cc` !important;
+  fill: currentcolor !important;
+}
+}
```

Stylelint (17.7.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)